### ISSUE TITLE
getting EMVS running on ROS noetic

### ISSUE THREAD

**Issue by Youras** [uzh-rpg/rpg_emvs#16]
#### Issue:

building in ros noetic throws an error which says that the pcl library needs to be built with c++14

#### Solution:

The solution is simply changing the **CMakeLists .txt** inside the **mapper_emvs** folder from c++**11** to c++**14**

from:
`set(CMAKE_CXX_FLAGS "-O3 -fopenmp -std=c++11 ${CMAKE_CXX_FLAGS}")`

to:
`set(CMAKE_CXX_FLAGS "-O3 -fopenmp -std=c++14 ${CMAKE_CXX_FLAGS}")`

Then most likely the library **cnpy** throws an error at runtime, because it was built in c++11 when cloning the dependencies, so we have to change the **CMakeLists.txt** here a well. Its in a seperate cnpy folder inside the src of your workspace.

basically we need to add this two lines after catkin_simple()

```
set(CMAKE_CXX_STANDARD 14)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

and then build again with

`catkin build mapper_emvs`

**Comment by guillermogb** [comment]
Thanks @Youras !

**Comment by guillermogb** [comment]
This lead to https://github.com/uzh-rpg/rpg_emvs/pull/17
